_replication/scheduling/advanced-cron-scheduling.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Advanced Replication Scheduling Using Cron Expressions
+permalink: /replication/replication-scheduling/advanced-cron-scheduling
+keywords: replicate, replication, replication frequency, frequency, anchor time, scheduling, schedule, interval, change replication time
+tags: [replication]
+
+summary: "The Advanced Scheduler feature allows you to specify granular start times for data extraction. Using cron expressions, you can specify the exact times, days of the week, or even days of the month data extraction should begin."
+type: "scheduling"
+toc: true
+weight: 4
+
+enterprise-cta:
+  title: "Need advanced scheduling?"
+  url: "?utm_medium=docs&utm_campaign=cron-scheduling"
+  copy: |
+    **Advanced replication scheduling is only available on an Enterprise plan**. Cron scheduling allows you to fine-tune an integration's replication schedule, ensuring you have the data you need when you need it. [Contact Stitch Sales for more info]({{ site.sales | append: page.enterprise-cta.url }}).
+
+---
+
+{% include enterprise-cta.html %}
+
+{{ page.summary }}
+
+**Note**: All replication scheduling methods (Replication Frequency, Anchor Scheduling, and Advanced Scheduling) define when data extractions begin. They do not control how long a replication job runs or when data is loaded into a destination.
+
+In this guide, we'll cover:
+
+- [The uses for Advanced Scheduling](#uses)
+- [The basics of cron scheduling](#basics)
+- [Examples of replication schedules using cron expressions](#examples)
+- [Limitations of cron scheduling in Stitch](#limitations)
+- [Troubleshooting validation errors](#troubleshooting)
+- [Additional cron resources](#resources)
+
+---
+
+## Uses for Advanced Scheduling {#uses}
+
+Using Advanced Scheduling, you can:
+
+1. **Run reports on a schedule**. For example: You need reports for a 9:00AM meeting you have every day. You could create a replication schedule that runs a replication job every night at 12:00AM, which would ensure fresh data is available in the morning when you need it. Reports could be scheduled to run at 7:00AM, giving you time before your meeting to review the data and resolve replication issues, should any arise.
```

Comment:
This is a great use case for anchor scheduling but not for cron scheduling. cron lets you express schedules that anchor can't capture (like every 15 minutes from 7 `7,22,37,52 * * * *`). I don't know how best to express that in the docs though but a customer reviewing both features would probably saying to themselves "huh. I'm pretty sure anchor does this too."

Reply:
True - what if this were to be changed to be like `You have a meeting every Wednesday` versus `every day`?

Reply:
Done.